x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -45,119 +51,102 @@ export interface IOverviewValues extends IOverviewServerData {
   flashMessages: IFlashMessagesProps;
 }
 
-interface IListenerParams {
-  actions: IOverviewActions;
-}
-
-interface IOverviewLogic extends IKeaLogic, IListenerParams {
-  values: IOverviewValues;
-}
```

Comment:
We can DRY these out to repeatable interfaces using [Typescript Generics](https://www.digitalocean.com/community/tutorials/typescript-generics-in-typescript)! 🎉 No file should need to define its own `IListenerParams` anymore - we just pass in `ISomeActions`/`ISomeValues` to the reusable defs at the bottom of the page